DEPENDENCIES

```diff
@@ -9,12 +9,13 @@ Prerequisites for building the OpenDataPlane (ODP) API
    automake
    autoconf
    libtool
+   pkg-config
 
    On Debian/Ubuntu systems:
-   $ sudo apt-get install automake autoconf libtool
+   $ sudo apt-get install automake autoconf libtool pkg-config
 
    On CentOS/RedHat/Fedora systems:
-   $ sudo yum install automake autoconf libtool
+   $ sudo yum install automake autoconf libtool pkgconfig
```

Comment:
Shouldn't this be pkg-config here as well?

Reply:
no. Check packages on CentOS/any other rpm distro.